kotlintest-assertions/src/commonMain/kotlin/io/kotlintest/properties/gens.kt

```diff
@@ -490,6 +490,39 @@ fun Gen.Companion.choose(min: Long, max: Long): Gen<Long> {
   }
 }
 
+/**
+ * Returns a stream of values based on weights:
+ *
+ * <pre>Gen.choose(1 to 'A', 2 to 'B')</pre> will generate 'A' 33% of the time
```

Comment:
I don't think IntelliJ likes <pre> or any other tag for Kotlin docs

Reply:
So, remove the pre tags?

Reply:
Or try to `ctrl + Q` it and see if it renders correctly